examples/exp_of_DCRNN/main.py

```diff
@@ -0,0 +1,63 @@
+import torch
+from dataset import PemsBayDatasetLoader
+from dcrnn import DCRNN
+
+
+feature_dim = int(len(dataset.features[0][0]) * len(dataset.features[0][0][0]))
+node_num = int(len(dataset.features[0]))
+
+split_train_test_ratio = 0.7
+
+#Load Dataset
+loader = PemsBayDatasetLoader()
+dataset = loader.get_dataset()
+
+#Split Train and Test
+train_dataset, test_dataset = dataset.features[:int(len(dataset.features) * split_train_test_ratio)] , dataset.features[int(len(dataset.features) * split_train_test_ratio):]
```

Comment:
there is a duplication of `int(len(dataset.features) * split_train_test_ratio)`. Please, consider the creation of a variable to store this value to avoid duplication and also to reduce this line size.
```suggestion
split_idx = int(len(dataset.features) * split_train_test_ratio)
train_dataset, test_dataset = dataset.features[:split_idx], dataset.features[split_idx:]
```